iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/ArmTree.java

```diff
@@ -39,6 +39,7 @@ enum Kind implements GrammarRuleKey {
     FILE(File.class),
     OUTPUT_DECLARATION(OutputDeclaration.class),
     PARAMETER_DECLARATION(ParameterDeclaration.class),
+    GROUP_RESOURCE_DECLARATION(GroupResourceDeclaration.class),
```

Comment:
When looking multiple times at the name maybe `ResourecGroupDeclaration` is more suitable. WDYT?